docs/Configuration-Settings.md

```diff
@@ -180,11 +180,26 @@ number.
     specified for `tachyon.master.hostname`.</td>
 </tr>
 <tr>
+  <td>tachyon.master.bind.host</td>
+  <td>0.0.0.0</td>
+  <td>The hostname Tachyon's master node binds to.</td>
+</tr>
+<tr>
   <td>tachyon.master.port</td>
   <td>19998</td>
   <td>The port Tachyon's master node runs on.</td>
 </tr>
 <tr>
+  <td>tachyon.master.web.hostname</td>
+  <td>localhost</td>
+  <td>The externally resolvable hostname of master's web service that a client uses to communicate with the service.</td>
+</tr>
+<tr>
+  <td>tachyon.master.web.bind.host</td>
+  <td>0.0.0.0</td>
+  <td>The hostname Tachyon's master's web server binds to.</td>
```

Comment:
Same here, is it possible for this value to be different from `tachyon.master.web.hostname` or `0.0.0.0`?